services/common/serv_helper.js

```diff
@@ -5,11 +5,111 @@
     console.log('-*-*- ServHelper: ' + str);
   }
 
+  var ACCESS_CTRL_FILE = 'acl.json';
+
+  function getACL(aUrl) {
+    function aclPathFromURL(aUrl) {
+      if (!aUrl) {
+        debug('Trying to recover acl file from no url');
+        return {};
+      }
+      // Error ctrl?
+      var url = new URL(aUrl);
+      var pathname = url.pathname;
+      return pathname.substring(0, pathname.lastIndexOf('/') + 1);
+    }
+    return new Promise((resolve, reject) => {
+      var xhr = new XMLHttpRequest();
+      xhr.open('GET', aclPathFromURL(aUrl) + ACCESS_CTRL_FILE, true);
+      xhr.responseType = 'json';
+
+      xhr.onerror = function(error) {
+        reject(error);
+      };
+      xhr.onload = function() {
+        if (xhr.response !== null) {
```

Comment:
I didn't do it because if I do it too this PR will be bigger than it will be advisable (In fact I think It's already too big), but you're right and I'll take care of this